.github/workflows/e2e.istio.yaml

```diff
@@ -0,0 +1,127 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  SKIP_TEST: true
+  ISTIO_VERSION: 1.7.1
+  TAG: ${{ github.sha }}
+  SCRIPTS_DIR: main/test/e2e-mesh/e2e-istio/scripts
+  SW_OAP_BASE_IMAGE: openjdk:8-jre-alpine
+
+jobs:
+  als:
```

Comment:
I think we need `als` tests for the main repo tag or tags? How about set up a `commit-id` matrix?